File: src/main/java/pl/dmichalski/algorithms/_1_sum_from_1_to_n/Runner.java
package pl.dmichalski.algorithms._1_sum_from_1_to_n;

public class Runner {

    public static void main(String[] args) {
        int n = 100;

        SumCounterService1 sumCounterService1 = new SumCounterService1();
        SumCounterService2 sumCounterService12 = new SumCounterService2();

        int sum1 = sumCounterService1.countSumFrom1ToN(n);
        int sum2 = sumCounterService12.countSumFrom1ToN(n);

        System.out.println("sum1 = " + sum1);
        System.out.println("sum2 = " + sum2);
    }

}

File: src/main/java/pl/dmichalski/algorithms/_1_sum_from_1_to_n/Runner.kt
package pl.dmichalski.algorithms._1_sum_from_1_to_n

/**
 * Write a function which takes n number and returns
 * sum from 1 to n inclusive.
 */
object Runner {

    @JvmStatic
    fun main(args: Array<String>) {
        val n = 100

        val sumCounterService1 = SumCounterService1()
        val sumCounterService12 = SumCounterService2()

        val sum1 = sumCounterService1.countSumFrom1ToN(n)
        val sum2 = sumCounterService12.countSumFrom1ToN(n)

        println("sum1 = $sum1")
        println("sum2 = $sum2")
    }
}
